bluetooth/src/commonMain/kotlin/UUID.kt

```diff
@@ -15,12 +15,19 @@
 
  */
 
+@file:JvmName("CommonUUID")
 package com.splendo.kaluga.bluetooth
 
+import kotlin.jvm.JvmName
+
 expect class UUID
 
 expect val UUID.uuidString: String
 
 expect fun uuidFrom(uuidString:String):UUID
 
 expect fun randomUUID():UUID
+
+fun uuidFromShort(uuidString: String): UUID = uuidFrom("0000$uuidString-0000-1000-8000-00805f9b34fb")
```

Comment:
I think it would be nice to specify why we use this particular values to complete the UUID, with some documentation.
In this case, this comes from Bluetooth specifications, and it would be great if you can mention it here and even better if you can find the link where is defined